examples/flower-in-30-minutes/tutorial.ipynb

```diff
@@ -826,23 +783,48 @@
    "execution_count": null,
    "metadata": {
     "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 508
+     "base_uri": "https://localhost:8080/"
     },
-    "outputId": "d8eab106-cee9-4266-9082-0944882cdba8"
+    "outputId": "9ad8dcea-8004-4c6e-a025-e168da636c88"
    },
    "outputs": [],
    "source": [
-    "print(f\"{history.metrics_centralized = }\")\n",
+    "from flwr.server import ServerApp, LegacyContext, ServerConfig\n",
+    "from flwr.server.workflow import DefaultWorkflow\n",
+    "from flwr.simulation import run_simulation\n",
+    "\n",
+    "server_app = ServerApp()\n",
+    "\n",
+    "\n",
+    "@server_app.main()\n",
+    "def main(driver, context):\n",
+    "    # Construct the LegacyContext\n",
+    "    context = LegacyContext(\n",
```

Comment:
I'm unsure if we want to show this way of constructing `ServerApp` in a tutorial that's designed for people using Flower for the first time. The topic of the `History` class is still tbd in the new flower working with `ClientApp` and `ServerApp`.

Reply:
@jafermarq should we just remove this? But I like that we show the graphs as a result for the FL simulation.

Reply:
deleted workflow and move it to custom strategy.